### ISSUE TITLE
Lock down more fmf id URL to git urls

### ISSUE THREAD

**Issue by thrix** [teemtee/tmt#1258]
See: https://github.com/teemtee/tmt/pull/1202/files#r896677537

Seems this could work:

`((git|ssh|http(s)?)|(git@[\w\.]+))(:(//)?)([\w\.@\:/\-~]+)(\.git)(/)?`

https://www.debuggex.com/r/H4kRw1G0YPyBFjfm

**Comment by lukaszachy** [comment]
Well, debuggex shows that _local_ git repositories are not accepted. Do we want that? While it doesn't make sense for "production" plans, for local it is very handy and current [definition](https://fmf.readthedocs.io/en/latest/concept.html#identifiers) of 'url' attribute accepts it by stating "Git repository containing the metadata tree. Use any format acceptable by the git clone command. "

**Comment by thrix** [comment]
yeah, with local it can basically become any string from validation point of view, so I guess, best we can do ...

**Comment by psss** [comment]
Ok, so nothing to be done, let's remove the todo links: #4149
